### ISSUE TITLE
Exception thrown on Delete requests without body parameters

### ISSUE THREAD

**Issue by khkramer** [zfcampus/zf-content-negotiation#34]
This happened when composer updated zf-content-negotiation.
It seems like it's trying to set the body parameters even if the delete resource doesn't require one.

```
Argument 1 passed to ZF\ContentNegotiation\ParameterDataContainer::setBodyParams() must be of the type array, null given
```

**Comment by eimanavicius** [comment]
I have the same problem, switched to "zfcampus/zf-content-negotiation": "1.0.3"

**Comment by Wilt** [comment]
Related to: https://github.com/zfcampus/zf-apigility/issues/101#issuecomment-95488109

**Comment by mbn18** [comment]
Same here, Fixed the issue by adding to compose (under require):
```"zfcampus/zf-content-negotiation": "1.0.3"```

**Comment by weierophinney** [comment]
This was fixed with zfcampus/zf-content-negotiation#35, which was released in v1.0.5 of that module. Updating that dependency will fix the issues.
